CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.12)
+project(aerosol-radiation)
+enable_language(Fortran)
```

Comment:
Maybe `enable_language(CXX)` here just for clarity.

Reply:
done!